dev/com.ibm.ws.transport.http2_fat/test-applications/H2FATDriver.war/src/http2/test/driver/war/servlets/H2FATDriverServlet.java

```diff
@@ -3379,8 +3379,14 @@ public void testHeaderFrameDependsOnItself(HttpServletRequest request,
         CountDownLatch blockUntilConnectionIsDone = new CountDownLatch(1);
         Http2Client h2Client = getDefaultH2Client(request, response, blockUntilConnectionIsDone);
 
-        FrameRstStream rstFrame = new FrameRstStream(3, PROTOCOL_ERROR, false);
-        h2Client.addExpectedFrame(rstFrame);
+        if (USING_NETTY) {
+            byte[] debugData = "HEADERS frame for stream 3 cannot depend on itself.".getBytes();
```

Comment:
Discuss since this is a change in behavior but one that matches more closely to the preservation of header tables